src/components/Overview/ImageGallery.jsx

```diff
@@ -1,7 +1,70 @@
 import React, {useEffect, useState} from 'react';
+import styled from 'styled-components';
+import { FaAngleLeft, FaAngleRight, FaAngleUp, FaAngleDown } from 'react-icons/fa';
 
-const ImageGallery = () => {
-  return (<div className='ImageGallery'>ImageGallery</div>)
+const Img = styled.img`
```

Comment:
I would recommend putting all your styles in one file and then importing them into the files you need - just makes for a cleaner look

Reply:
So can you import a styled component? I'm using 2 different method to style. Not sure if I'm able to put everything in one file